ui/src/app/applications/components/application-pod-view/pod-view.tsx

```diff
@@ -108,19 +136,6 @@ export class PodView extends React.Component<PodViewProps> {
                                                                 </div>
                                                             )}
                                                         </div>
-                                                        <div style={{marginLeft: 'auto'}}>
-                                                            {group.renderMenu && (
-                                                                <DropDown
-                                                                    isMenu={true}
-                                                                    anchor={() => (
-                                                                        <button className='argo-button argo-button--light argo-button--lg argo-button--short'>
-                                                                            <i className='fa fa-ellipsis-v' />
-                                                                        </button>
-                                                                    )}>
-                                                                    {() => group.renderMenu()}
-                                                                </DropDown>
-                                                            )}
-                                                        </div>
                                                     </div>
```

Comment:
@rbreeze, how about the custom resource actions written in  Lua?  Will they not be made available if the menu is removed?

Reply:
That's a great point @keithchong. Maybe we should leave the ellipsis menu in that case, but if possible, remove redundant items. It would be even better if we could show the ellipsis menu only if that resource has custom resource actions available.